src/Components/ArticleDetails.jsx

```diff
@@ -93,11 +103,15 @@ function ArticleDetails() {
         article_id={article_id}
         onCommentAdded={handleCommentAdded}
       />
-      <p>Comments: {article.comment_count}</p>
+      <p>Comments: {commentCount}</p>
       <div className="comments-section">
         {comments.length > 0 ? (
           comments.map((comment) => (
-            <CommentCard key={comment.comment_id} comment={comment} />
+            <CommentCard
+              key={comment.comment_id}
+              comment={comment}
+              onDelete={handleDelete}
+            />
```

Comment:
I like that you passed down the handleDelete function so that you can update comment count in this component.